.circleci/config.yml

```diff
@@ -390,13 +385,40 @@ commands:
             git status
             gh pr create --fill --base main --head $CIRCLE_BRANCH
 
+  build-and-push-docker-image:
+    parameters:
+      image-name:
+        type: string
+    steps:
+      - run:
+          name: Build  Docker Image
+          command: docker build -f BuilderImage.Dockerfile . -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-${CIRCLE_SHA1} -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >> -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-latest
+      - run:
+          name: Push Builder Docker Image
+          command: |
+            aws ecr get-login-password --region ${AWS_REGION} | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-${CIRCLE_SHA1}
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-latest
+
+  docker-deploy-image:
+    steps:
+      - configure
+      - setup_remote_docker:
+          version: 20.10.14
+          docker_layer_caching: true
+      - build-and-push-docker-image:
+          image-name: test
```

Comment:
This name should be more descriptive of what the image is. Naming an image test won't tell the user what it is used for,

Reply:
Renamed the image to node_v16_16

Reply:
Thats still not a good name for this image. The image name should describe what the image is for. In this case, its a builder for the NJ BFS Navigator project